app/styles/components/task-list-cards.scss

```diff
@@ -1,35 +1,57 @@
 .task-list-container {
+  box-sizing: border-box;
+  display: inline-block;
+  height: 100%;
+  margin-left: 7px;
+  white-space: nowrap;
+  width: 280px;
+
   &:first-child {
-    .task-list {
-      margin-left: 10px;
-    }
+    margin-left: 10px;
   }
 
   &:last-child {
-    .task-list {
-      margin-right: 10px;
-    }
+    margin-right: 10px;
   }
 }
 
 .task-list {
   border: 1px solid #DFDFDF;
   background-color: #F7F7F7;
   border-radius: 4px;
-  margin-left: 7px;
+  display: flex;
+  flex-direction: column;
   max-height: 100%;
-  overflow-x: hidden;
-  overflow-y: scroll;
-  padding: 8px 6px;
+  position: relative;
   white-space: normal;
-  width: 280px;
 
   &__name {
     font-weight: 600;
-    margin-bottom: 10px;
+    padding: 8px 6px;
   }
 }
 
 .task-list-cards {
-  height: 100%;
+  margin: 0 3px;
+  min-height: 10px;
+  overflow-x: hidden;
+  overflow-y: auto;
+  padding: 0 3px;
+  white-space: normal;
```

Comment:
I wonder if this `white-space` declarations are needed?

Reply:
Needed due to parent/child `white-space` declarations.